core/src/test/java/feign/ResponseTest.java

```diff
@@ -31,41 +31,42 @@
 
   @Test
   public void reasonPhraseIsOptional() {
-    Response response = Response.create(200, null /* reason phrase */, Collections.
-        <String, Collection<String>>emptyMap(), new byte[0]);
+    Response response = Response.builder()
+            .status(200)
+            .headers(Collections.<String, Collection<String>>emptyMap())
+            .body(new byte[0])
+            .build();
 
     assertThat(response.reason()).isNull();
     assertThat(response.toString()).isEqualTo("HTTP/1.1 200\n\n");
   }
 
   @Test
-  public void lowerCasesNamesOfHeaders() {
```

Comment:
The coverage provided by this class is also handled in 'canAccessHeadersCaseInsensitively'